hazelcast/src/main/java/com/hazelcast/partition/impl/InternalPartitionServiceImpl.java

```diff
@@ -1530,6 +1540,37 @@ public void onPartitionLost(InternalPartitionLostEvent event) {
         eventService.publishEvent(SERVICE_NAME, registrations, partitionLostEvent, event.getPartitionId());
     }
 
+    /**
+     * @return copy of ongoing replica-sync operations
+     */
+    public List<ReplicaSyncInfo> getOngoingReplicaSyncRequests() {
+        final int length = replicaSyncRequests.length();
```

Comment:
Would it not be better to have this method to have this method exposed on the InternalPartitionService?